client/src/components/PacmanGame/gameCore.js

```diff
@@ -1,19 +1,13 @@
-export const initGridState = (rows, cols) => {
-  // const column = new Array(cols).fill(0);
-  // return new Array(rows).fill(column);
-  return Array(rows).fill().map(() => Array(cols).fill(0))
-};
-
-
-
-
+export // const column = new Array(cols).fill(0);
+// return new Array(rows).fill(column);
+const initGridState = (rows, cols) => Array(rows).fill().map(() => Array(cols).fill(0));
 export const getFoods = () => [[15, 9], [14, 9], [13, 9], [12, 9], [11, 9], [10, 9]];
 
-const wallGenerator = numberofCells => {
+const wallGenerator = (numberofCells) => {
```

Comment:
With this you just have to create a n*n wall right? You can resolve it simply with n^2 complexity. Are you trying to reduce the complexity?

Reply:
This will be used in the future to create a complex wall pattern. Currently, it is rendering an arbitrary pattern.
Wall is in blue:
![image](https://user-images.githubusercontent.com/10586637/54321913-009e1e80-4619-11e9-89e8-fd6addf9bd48.png)